Rubberduck.Core/UI/Settings/GeneralSettings.xaml

```diff
@@ -131,6 +131,8 @@
                               IsChecked="{Binding CheckVersionAtStartup}" />
                     <CheckBox Margin="5,0,0,5" Content="{Resx ResxName=Rubberduck.Resources.RubberduckUI, Key=GeneralSettings_CompileBeforeParse}"
                               IsChecked="{Binding CompileBeforeParse}" />
+                    <CheckBox Margin="5,0,0,5" Content="{Resx ResxName=Rubberduck.Resources.RubberduckUI, Key=GeneralSettings_SetDpiUnaware}"
+                              IsChecked="{Binding SetDpiUnaware}" />
```

Comment:
I'd consider adding a `Enabled` binding to light this up only if it's Windows 8.1 or later (or whatever the DLL we're calling shipped with).

Reply:
Done.